proto/umee/incentive/v1/tx.proto

```diff
@@ -93,28 +93,28 @@ message MsgGovSetParams {
 // MsgGovSetParamsResponse defines the Msg/SetParams response type.
 message MsgGovSetParamsResponse {}
 
-// MsgGovCreateProgram is used by governance to create an incentive program. There are two
-// funding scenarios, depending on from_community_fund.
-// If it is true,the program's total rewards will be automatically withdrawn from
+// MsgGovCreatePrograms is used by governance to create one or more incentive programs.
+// There are two funding scenarios, depending on from_community_fund.
+// If it is true,the programs' total rewards will be automatically withdrawn from
 // the (parameter) community_fund_address to the incentive module account when this
 // message is passed. (Insufficient funds cause the parameter to be treated as false.)
-// If it is false, a MsgSponsor fundign the program's full amount must be submitted
+// If it is false, a MsgSponsor funding each program's full amount must be submitted
 // after this message passes, but before the program's start_time, or the program
 // will be cancelled when it would otherwise start.
-message MsgGovCreateProgram {
+message MsgGovCreatePrograms {
   option (gogoproto.equal)           = true;
   option (gogoproto.goproto_getters) = false;
   option (cosmos.msg.v1.signer)      = "authority";
 
   // authority must be the address of the governance account.
-  string           authority   = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
-  string           title       = 2;
-  string           description = 3;
-  IncentiveProgram program     = 4 [(gogoproto.nullable) = false];
+  string                    authority   = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string                    title       = 2;
+  string                    description = 3;
+  repeated IncentiveProgram programs    = 4 [(gogoproto.nullable) = false];
```

Comment:
What's the reason to change the proposal type to vote for multiple programs at once?

Reply:
It may be that we want to propose (or renew) programs in batches when they're under the same theme. This allows that option without spamming individual gov proposals